vendor/tektelic/encoder_home_sensor.js

```diff
@@ -0,0 +1,700 @@
+function Encoder(data) {
```

Comment:
This is the V2 format, which is supported in The Things Stack, but it is not supported in the Device Repository. Please use input and output objects; https://thethingsstack.io/integrations/payload-formatters/javascript/